django/conf/project_template/project_name/settings.py-tpl

```diff
@@ -114,6 +114,14 @@ USE_L10N = True
 USE_TZ = True
 
 
+# Security settings
+SECURE_CONTENT_TYPE_NOSNIFF = True
+
+SECURE_BROWSER_XSS_FILTER = True
+
+X_FRAME_OPTIONS = 'DENY'
```

Comment:
This probably also needs doc adjustments here https://docs.djangoproject.com/en/2.2/ref/clickjacking/#setting-x-frame-options-for-all-responses